uni/lib/model/entities/bug_report.dart

```diff
@@ -1,19 +1,38 @@
+// Stores information about Bug Report
+import 'package:json_annotation/json_annotation.dart';
 import 'package:tuple/tuple.dart';
 
+part 'bug_report.g.dart';
+
+class TupleConverter extends JsonConverter<Tuple2<String, String>?, String?> {
+  const TupleConverter();
+
+  @override
+  Tuple2<String, String>? fromJson(String? json) {
+    if (json == null) return null;
+    return Tuple2<String, String>('', json);
+  }
+
+  @override
+  String? toJson(Tuple2<String, String>? object) {
+    if (object == null) return null;
+    return object.item2;
+  }
+}
```

Comment:
This `TupleConverter` is suited only for the logic of this `BugReport` DTO. Should we call it something else, perhaps private?

Reply:
I don't this is a good idea, the generated files aren't capable of accessing private fields or classes.